rl4co/models/zoo/pomo/model.py

```diff
@@ -46,7 +55,13 @@ def __init__(
         self.save_hyperparameters(logger=False)
 
         if policy is None:
-            policy = POMOPolicy(env.name, **policy_kwargs)
+            default_values = {
+                "num_encoder_layers": 6,
+                "normalization": "instance",
+                "use_graph_context": False,
+            }
+            default_values.update(policy_kwargs)
```

Comment:
default_values is never used

Reply:
Great catch! We need to add something like `policy_kwargs = default_values.update(policy_kwargs)` to overwrite them

Reply:
Yes (keep in mind that update() returns None though)